src/core/api/repository.go

```diff
@@ -1022,33 +1022,6 @@ func (ra *RepositoryAPI) VulnerabilityDetails() {
 	ra.ServeJSON()
 }
 
-// ScanAll handles the api to scan all images on Harbor.
-func (ra *RepositoryAPI) ScanAll() {
```

Comment:
I do have some concern if we remove this API in a sudden, bottom-line we need to communicate to the community, and document this as a breaking change in release note.